modules/core/04-channel/v2/keeper/events.go

```diff
@@ -11,6 +11,11 @@ func EmitSendPacketEvents(ctx context.Context, packet channeltypesv2.Packet) {
 	// TODO: https://github.com/cosmos/ibc-go/issues/7386
 }
 
+// EmitTimeoutPacketEvents emits events for the TimeoutPacket handler.
+func EmitTimeoutPacketEvents(ctx context.Context, packet channeltypesv2.Packet) {
+	// TODO: https://github.com/cosmos/ibc-go/issues/7386
```

Comment:
just noticed this issue is for SendPacket explicitly, I think we can update the issue to be a general tracker for the event changes.